src/components/Card/src/CardActions/CardAction.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import MaterialCardActions from "@material-ui/core/CardActions";
+import BaseProps from "@gemeente-denhaag/baseprops";
+import "./card-actions.css";
+import "./mui-override.css";
+import { StylesProvider } from "@material-ui/core/styles";
+import { cardActionClasses } from "./bem-mapping";
+
+export interface CardActionProps extends BaseProps {
+  /**
+   * If `true`, the actions do not have additional margin.
+   */
+  disableSpacing?: boolean;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const CardActions: React.FC<CardActionProps> = (props: CardActionProps) => {
+  return (
+    <StylesProvider injectFirst>
+      <MaterialCardActions classes={cardActionClasses} {...props}>{props.children}</MaterialCardActions>
```

Comment:
`{...props}`
We don't want to blindly pass props to the underlying material UI component. This potentially exposes MUI API which we don't wish to support. See this discussion for more details: https://github.com/Gemeente-DenHaag/denhaag-component-library/pull/133#discussion_r637213463

Reply:
Fixed